src/requirements/data/majors/phys.ts

```diff
@@ -0,0 +1,121 @@
+    description: 'At least three credits of laboratory work.',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19990',
+    checker: includesWithSingleRequirement(
+      'PHYS 3310',
+      'PHYS 3330',
+      'PHYS 3360',
+      'PHYS 4410',
+      'AEP 2640',
+      'ASTRO 4410',
+      'BEE 4500'
+    ),
+    fulfilledBy: 'credits',
+    perSlotMinCount: [3],
+  },
+  {
+    name: 'Physics Intermediate Courses',
+    description:
+      'An intermediate course in analytical mechanics and advanced electricity & magnetism.',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19990',
+    checker: includesWithSubRequirements(['PHYS 3318'], ['PHYS 3327']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1],
+    slotNames: ['PHYS 3318', 'PHYS 3327'],
+  },
+  {
+    name: 'Concentration',
+    description: 'The Physics Department offers two approaches to the major.',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=41&poid=19990',
+    fulfilledBy: 'toggleable',
+    checkerWarning: 'We do not check that courses are approved by the major faculty advisor.',
+    fulfillmentOptions: {
+      'Concentration "inside" Physics': {
+        description:
+          'The concentration within physics ("inside concentration") is the principal path to ' +
+          'professional or graduate work in physics and closely related fields, and is also the best choice ' +
+          'for students who wish to obtain maximum benefit from rigorous studies in physics. The inside ' +
+          'concentration consists of the core physics courses plus electives taken within the Physics Department.',
+        counting: 'credits',
+        // "Other courses approved by the director of undergraduate studies"
+        // do not pass this checker, and must be manually overwritten by the user.
+        // "If this activity is done as an independent project, PHYS 4490, up to 4 credits can be applied toward the concentration"
+        // must also be manually overwritten by the user.
+        // TODO implement as compound requirement:
+        // at least two MATH classes at 3000+ level
+        // must include PHYS 4230, PHYS 4410, and another lab course listed for the core (allow double-counting)
+        checker: [
+          (course: Course): boolean =>
+            ifCodeMatch(course.subject, 'PHYS') &&
+            !(ifCodeMatch(course.catalogNbr, '1***') || ifCodeMatch(course.catalogNbr, '2***')),
+        ],
+        perSlotMinCount: [15],
+      },
+      'Concentration "outside" Physics': {
+        description:
+          'The concentration outside physics ("outside concentration") provides more flexibility ' +
+          'for those want to develop skills in physics but whose career interests lie elsewhere.',
+        counting: 'credits',
+        // This is essentially self-check; courses must be manually overwritten by the user.
+        checker: [(): boolean => false],
```

Comment:
You should make `checker: [(): boolean => true],` so that a course could be applied to it. Currently, I can't add any course

Reply:
Then any course would be able to fulfill the requirement, right? I thought we wanted requirements to be as limiting as possible, so we wouldn't accidentally be telling the user that they finished a requirement when they really didn't.

My idea was to make no courses apply to it, and the user would have to override to apply anything to it. If any course fulfilled the requirement, then overriding wouldn't be useful here. I originally thought of explicitly classifying this as self-check, but didn't because we currently don't support self-check within toggleable requirements.

Reply:
Yeah I see your point. I wonder if we can update the checker so that it doesn't include courses from PHYS. Would that be a safe assumption to make for the outside concentration? Since we have a `checkerWarning` for this requirement, we're implicitly treating this concentration as having as weak but accurate of a checker as possible so that the user can essentially use whatever course to fulfill it. @einc what are your thoughts?

Reply:
Yeah I see your point. I wonder if we can update the checker so that it doesn't include courses from PHYS. Would that be a safe assumption to make for the outside concentration? Since we have a `checkerWarning` for this requirement, we're implicitly treating this concentration as having as weak but accurate of a checker as possible so that the user can essentially use whatever course to fulfill it. @einc what are your thoughts?

Reply:
I think having any courses outside Physics would be good for now since yea we have the checkerWarning!